### PR TITLE
Add Component Styling

### PR DESCRIPTION
**Pull request by cancelei**
Hello Reviewer,

This is part 2 of the Math Magicians Project.

Highlights:
- 'CSS Styling Implementation' using only JSX.
- Updated Readme File.
- Linters Passing.

Good review.

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    display: 'grid',
+    gridTemplateColumns: 'repeat(4, 1fr)',
+  };
+
+  return (
+    <div style={calculatorStyle}>
+      <input placeholder="0" style={displayStyle} />
+
+      <div style={buttonParent}>
+        <button type="button" style={buttonStyle}>
```
Comment (iambenkis) [inline src/Components/Calculator.js:40]: Keep doing the great work 👍

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @cancelei 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Component  directory
✔️ `Calculator.js` component has **functional** component 



## Suggested changes

- [x] Good job including live demo link and video presentation link to your README documentation , 👏 👏 👏 , However the links attached are not related to the project you made, Kindly replace them if they exist or use the `N/A` if not 😄 

Capture
 

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/Components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          .
+        </button>
+        <button type="button" style={{ ...buttonStyle, backgroundColor: 'orange' }}>
+          =
+        </button>
+      </div>
+    </div>
+  );
+}
```
Comment (roshan-bajgain) [inline src/Components/Calculator.js:100]: - [ ] [Optional] According to the requirement Calculator.js should have at least 1 child component. please implement it.
- [ ] [Optional] I would suggest you please create a different file for styling and it looks more readable and clean.

**Review by roshan-bajgain — APPROVED** [review]
Hi @cancelei, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job ✔️ 

## Optional suggestions
- [ ] I saw N/A in you contribution section which is not professional. So I would suggest you please provide `issue link` and leave a message. `Contributions, issues, and feature requests are welcome!` and provide the link to `Issue page` It make more professional README file.
- [ ] Same thing to your `Show you Support` in your README file. please provide `If you like this project kindly leave a star!` which is far better than `N/A`.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**